tasks/task383_matres_classification.json

```diff
@@ -0,0 +1,11558 @@
+{
+    "Contributors": [
+        "Yeganeh Kordi"
+    ],
+    "Source": [
+        "MATRES (http://cogcomp.org/papers/NingWuRo18.pdf), (https://github.com/CogComp/MATRES)"
+    ],
+    "Categories": [
+        "Classification" 
+    ],
+    "Definition": "You will be given a question and a context separated with a \n, and you have to answer that the given verb can be anchored in time or not. We say a verb can be anchored in the real timeline if and only if an verb happened in the past or is happening now or is unconditionally planned to happen in the future. If the verb is not anchorable, provide more detail by choosing one of the other labels. If verb is about a intention, wish or opinion in the future but it's not unconditional use No, It's intention/wish/opinion, if verb implies a condition or hypothetical choose No, It's hypothetical/condition this verbs usually come after if, choose No, It's negation if verb is about an event that is not going to happen, if the verb doesn't limited to a specific time choose No, It's abstract/non-specific, If a verb always happens use No, It's static and if it's about a repetitive event use No, It's recurrent. And then classify your answers into \"Yes, it is anchorable\", \"No, it is intention/wish/opinion\", \"No, it is hypothetical/condition\", \"No, it is negation\", \"No, it is abstract/nonspecific\", \"No, it is static\" and \"No, it is recurrent\".",
```

Comment:
I am still a bit fuzzy on the definition. Am I correct that this is a whole sentence? In which case, it is a very complex and long sentence (besides its grammatical issues). In that case, it might be helpful to try to break it into several simpler sentences. 

> If verb is about a intention, wish or opinion in the future but it's not unconditional use No, It's intention/wish/opinion, if verb implies a condition or hypothetical choose No, It's hypothetical/condition this verbs usually come after if, choose No, It's negation if verb is about an event that is not going to happen, if the verb doesn't limited to a specific time choose No, It's abstract/non-specific, If a verb always happens use No, It's static and if it's about a repetitive event use No, It's recurrent.

Reply:
Sorry, it wasn't supposed to be one sentence. I updated the definition and hope it's better now.